deploy/infrastructure/modules/terraform-google-dss/README.md

```diff
@@ -0,0 +1,71 @@
+# terraform-google-dss
+
+This terraform module creates a Kubernetes cluster in Google Cloud Engine and generates 
+the tanka files to deploy a DSS instance.
+
+## Getting started
+
+### Prerequisites
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Google Cloud Engine](./README.md#google-cloud-engine)
+    
+#### Google Cloud Engine
+
+1. Install and initialize [Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+    1. Confirm successful installation with `gcloud version`.
+2. Check that the DSS project is correctly selected: gcloud config list project
+    1. Set another one if needed using: `gcloud config set project $GOOGLE_PROJECT_NAME`
+3. Enable the following API using [Google Cloud CLI](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud):
+    1. `container.googleapis.com`
+    2. If you want to manage DNS entries with terraform: `dns.googleapis.com`
+4. Install the auth plugin to connect to kubernetes: `gcloud components install gke-gcloud-auth-plugin`
+5. Run `gcloud auth application-default login` to generate credentials to call Google Cloud Platform APIs.
+    1. If the result of performing the authorization indicates 404 in the browser, check whether a local dummy-oauth instance is running (using port 8085).  Stop the dummy-oauth instance if it is running.
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named for instance `terraform-google-dss-dev`.
+2. Copy main.tf, output.tf and variables.tf to the new folder. (Note that the modules can be added to existing projects)
+3. Copy `terraform.dev.example.tfvars` rename to `terraform.tfvars` and set the variables according to your environment. See [TFVARS.md](TFVARS.md) for variables descriptions.
+4. Check that your new directory contains the following files:
+   - main.tf
+   - output.tf
+   - terraform.tfvars
+   - variables.tf
+5. In the new directory (ie /deploy/infrastructure/personal/terraform-google-dss-dev), initialize terraform: `terraform init`.
+6. Run `terraform plan` to check that the configuration is valid. It will display the resources which will be provisioned.
+7. Run `terraform apply` to deploy the cluster. (This operation may take up to 15 min.)
+8. Configure the DNS resolution to the public ip addresses. DNS entries can be either managed manually or 
+handled by terraform depending on the cloud provider. See [DNS](DNS.md) for details.
+
+## Deployment of the DSS services
+
+During the successful run, the terraform job has created a new [workspace](../../../../build/workspace/)
+for the new cluster.
+
+It contains scripts to operate the cluster and setup the services.
+
+1. Go to `/build/workspace/${cluster_context}`. Run `terraform output` to find `cluster_context` value. 
```

Comment:
It seems like they don't need to run `terraform output` to determine `${cluster_context}` if they already know `${cluster_context}` in order to get to the folder in which `terraform output` needs to run in order to determine `${cluster_context}`.

Ideally, I think we'd still give users an easier way to determine `${cluster_context}`, but for now we could note on line 47 that the new folder created in the workspace folder is named `${cluster_context}`

Reply:
Please see latest commit.

Reply:
Looks good :)